### DISCUSSION TITLE
Multi-party threshold signing

### DISCUSSION THREAD

**Discussion by torvald2** [trustwallet/wallet-core#2694] [Q&A]
Hello. I`m researching how to use multi party signing for trust wallet. 
But have no ideas how to use it with wallet core api. 
There is good library https://github.com/taurusgroup/multi-party-sig for golang and I`m trying to find methods for create wallet address and signing transactions.
Is there gaoling examples for use external signing for this cases?

**Comment by hewigovens** [comment]
@catenocrypt maybe you have more insights on this?

**Comment by optout21** [comment]
From the point of view of wallet core the signature has to be provided, wallet core does not know the details how the signature is derived.
There is a GO example for external signature workflow, here:
https://github.com/trustwallet/wallet-core/blob/master/samples/go/sample/external_signing.go
